app/assets/javascripts/petition-widget-builder.js

```diff
@@ -0,0 +1,103 @@
+(function($) {
+
+  // This is not good, but I would not like to make a request to fetch this HTML
+  var template = [
+    "<div class='petition-widget-builder'>",
+    "  <h2> Incorporar petição </h2>",
+    "  <i class='close icon-close'></i>",
```

Comment:
On the layout the header looks a little different, the title is `40px` away from the top margin and this button is `38px` far from the right and top margins.